### DISCUSSION TITLE
Adding custom data to the right of the plot

### DISCUSSION THREAD

**Discussion by juferban** [PoisonAlien/maftools#1052] [Ideas]
Hi,

I was trying to create a oncoplot where patients are divided in two groups. At the right of the oncoplot I get the percent of patients with gene alteration. It would be great if we could add the option to add a different set of values there. I know the rightBarData allows to define another set of custom numeric values but in my case I want to substitute the alteration percent across all patients for a table where I show the percent of alteration in each of the two patient groups. 
So basically my request is to be able to either use rightBarData or the showPCT to incorporate information that is not necesarily numeric. In my case I tried to follow the same approach as generating custom data for the rightBarData and generated a data.frame with two columns, one column for the Hugo_Symbol and the other column containing the frequency of alteration for the two patient groups separated by a colon e.g. MYC   30.1% : 10.5%

**Comment by PoisonAlien** [comment]
Hi,

Sorry for not getting back to you earlier. I somehow overlooked the issue. 
It's a nice idea, and I will see what I can do.
